src/components/NftCard/Members.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+
+import {
+  AvatarGroupWrapper,
+  AvatarWrapper,
+  Caption,
+  MembersWrapper,
+  StyledAvatar,
+  StyledAvatarGroup,
+} from '@/components/NftCard/NftCard.styles'
+import { Text } from '@/components/Text'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+import { cVar } from '@/styles'
+
+export type Member = {
+  assetUrl?: string | null
+  name: string
+}
+
+type MembersProps =
+  | {
+      caption: 'Creator' | 'Owner'
+      members: Member
+    }
+  | { caption: 'Supporters'; members: Member[] }
+
+export const Members: React.FC<MembersProps> = ({ caption, members }) => {
+  const avatars =
+    members && Array.isArray(members)
```

Comment:
we used `Array.isArray(members)` two times. Let's store this boolean in a variable maybe?